### PR TITLE
[BUGFIX] #71: Fix tsconfig includes

### LINKED ISSUE #97: RTE.default.proc.overruleMode
https://github.com/netresearch/t3x-rte_ckeditor_image/blob/28e3ba729474a9a2b99afea0732994942638fc4d/ext_localconf.php#L9

https://github.com/netresearch/t3x-rte_ckeditor_image/issues/75#issuecomment-671932654

### PR DESCRIPTION
**Pull request by mcmulman**
Resolve #97

### CODE REVIEW

File: Resources/Public/JavaScript/Plugins/typo3image.js [hunk 1]
```diff
@@ -286,13 +286,14 @@
...
-
-        require(['TYPO3/CMS/Backend/Modal'], function (Modal) {
-            var $modal = Modal.advanced({
-                type: Modal.types.iframe,
-                title: editor.lang.common.image,
-                content: url,
+            deferred = $.Deferred(),
+            $modal;
+
+    require(['TYPO3/CMS/Backend/Modal'], function (Modal) {
```
Comment (CybotTM) [inline Resources/Public/JavaScript/Plugins/typo3image.js:292]: it seems there are unrelated white space changes - beside this, looks fine for me.

File: composer.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
 	"homepage": "https://www.netresearch.de",
 	"license": ["GPL-2.0-or-later"],
 	"require": {
-		"typo3/cms-core": "^10.4"
+		"typo3/cms-core": "^10.2"
```
Comment (CybotTM) [inline composer.json:8]: Version changes seems unrelated to "[BUGFIX] #71: Fix tsconfig includes #99" for me.

**Review by CybotTM — CHANGES_REQUESTED** [review]
"[BUGFIX] #71: Fix tsconfig includes #99"

How is this relaetd to #71?

File: composer.json (hunk 2 above)
Comment (mcmulman) [reply composer.json]: Right, removed.

File: Resources/Public/JavaScript/Plugins/typo3image.js (hunk 1 above)
Comment (mcmulman) [reply Resources/Public/JavaScript/Plugins/typo3image.js]: Right, removed.

**Pull request merged**